spdx-utils/src/main/kotlin/model/SpdxPackage.kt

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (C) 2017-2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.spdx.model
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.ossreviewtoolkit.spdx.isSpdxExpressionOrNotPresent
+
+/**
+ * Describes a software package.
+ */
+data class SpdxPackage(
+    /**
+     * A unique identifier for this [SpdxPackage] within a SPDX document.
+     */
+    @JsonProperty("SPDXID")
+    val spdxId: String,
+
+    /**
+     * The [SpdxAnnotation]s for the package.
+     */
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    val annotations: List<SpdxAnnotation> = emptyList(),
+
+    /**
+     * Acknowledgements for the package that may be required to be communicated in some contexts. This is not meant to
+     * include the package's actual complete license text, and may include copyright notices. The SPDX data creator
+     * may use this field to record other acknowledgements, such as particular clauses from license texts, which may be
+     * necessary or desirable to reproduce.
+     */
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    val attributionTexts: List<String> = emptyList(),
+
+    /**
+     * Checksums of the package.
+     */
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    val checksums: List<SpdxChecksum> = emptyList(),
+
+    /**
+     * Any general comments about the package.
+     */
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    val comment: String = "",
+
+    /**
+     * A text relating to a copyright notice, even if not complete. To represent a not present value
+     * [SpdxConstants.NONE] or [SpdxConstants.NOASSERTION] must be used.
+     */
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    val copyrightText: String = "",
```

Comment:
IMO we shouldn't assign an empty default value if this must not be blank (according to the check in the constructor).

Reply:
Done.